### ISSUE TITLE
DEV: An error log on Android and IOS when going to distance rate page

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#50960]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:**  dev
**Reproducible in production?:** dev
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @DylanDylann 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1729075718314489

## Action Performed:

1. Go to workspace
2. Go enable distance rate feature
3. Go to distance rate page
4. Click on any distance rate

## Expected Result:
No error logs
## Actual Result:
Error logs appears
## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [x] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos
![Screenshot 2024-10-16 at 17 42 21](https://github.com/user-attachments/assets/6761b585-ff3d-489b-a16b-fceb5872ab27)

https://github.com/user-attachments/assets/31f463e1-4de8-4052-8c1a-1cd79a1cf36c

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by dylanexpensify** [comment]
Hey @kadiealexander! I'm heading out on parental leave so reassigning this! TY! 🙇‍♂️

**Comment by kadiealexander** [comment]
Posted about this [here](https://expensify.slack.com/archives/C06ML6X0W9L/p1729571796245119).

**Comment by kadiealexander** [comment]
Bumped my Slack post.

**Comment by kadiealexander** [comment]
Reassigning as I'm OOO for the next two weeks, thanks Corinne!

**Comment by OfstadC** [comment]
Not sure where this is at - but it looks like we are waiting on proposals - is that correct @dukenv0307 ? 😃

**Comment by marcochavezf** [comment]
Yeah, I agree that this is a BE bug. I think it's low priority though

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
Not overdue

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
Moving it to weekly, feel free to change it if disagree

**Comment by marcochavezf** [comment]
No update, prioritizing other dailies

**Comment by marcochavezf** [comment]
Created a [PR](https://github.com/Expensify/Web-Expensify/pull/44690) to set an empty object in the db when the categories are removed

**Comment by marcochavezf** [comment]
No update, catching up

**Comment by marcochavezf** [comment]
No update, still catching up with a few other higher priorities

**Comment by marcochavezf** [comment]
No updates, focused on #stability and #convert issues for now

**Comment by marcochavezf** [comment]
No updates, still focused on other high priority issues

**Comment by marcochavezf** [comment]
Sorry no update here, focused on other issues for now

**Comment by marcochavezf** [comment]
Same

**Comment by marcochavezf** [comment]
I'm going to move it to monthly since this only impacts developers. Feel free to change it if you disagree

**Comment by marcochavezf** [comment]
No update focused on other high priorities before heading OOO

**Comment by marcochavezf** [comment]
No update, focused on other higher priorities

**Comment by marcochavezf** [comment]
Closing it for now, focused on phase 6 of Concierge AI
